src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -52,6 +52,14 @@ MACRO(OPENMVG_REPORT_NOT_FOUND REASON_MSG)
   RETURN()
 ENDMACRO(OPENMVG_REPORT_NOT_FOUND)
 
+# whether it is built with CCTags
+set(OpenMVG_USE_CCTAG @OpenMVG_USE_CCTAG@)
+
+if(OpenMVG_USE_CCTAG)
+  set(CCTag_DIR @CCTag_DIR@)
```

Comment:
I'm not sure it's useful to deal with these border cases that will probably make linking errors at the end.